ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerProxy.java

```diff
@@ -346,6 +350,7 @@ public void close() {
         LOG.warn(getId() + ": Failed to close " + SupportedDataStreamType.NETTY + " server", ignored);
       }
     });
+    pauseMonitor.stop();
```

Comment:
Do we need try catch ?  so that close can continue when pauseMonitor.stop() throw exception.

Reply:
I think, this is the last step in the proxy.close() call. The exception should be thrown.